### PR TITLE
Task-35726: Hide change password from settings for SSO users

### PR DESCRIPTION
**Pull request by azayati**
Hide security setting from settings for SSO users

### CODE REVIEW

File: webapp/pom.xml [hunk 1]
```diff
@@ -19,6 +19,14 @@
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
+  <dependencies>
```
Comment (azayati) [inline webapp/pom.xml]: Put this block after modules, why not provided scope ?

**Review by azayati — COMMENTED** [review]
@AzmiTouil See review

File: webapp/pom.xml (hunk 1 above)
Comment (AzmiTouil) [reply webapp/pom.xml]: Done !

**Comment by azayati** [comment]
@AzmiTouil PR approved, merge waiting after cherry pick on FT branch and getting the GO from QA
Please add PR description and approve it.

**Review by AzmiTouil — APPROVED** [review]
LGTM

**Pull request merged**